webapp/tests/base.py

```diff
@@ -0,0 +1,8 @@
+from django.test import TestCase
+from graphite.worker_pool.pool import stop_pool
+
+
+class BaseTestCase(TestCase):
```

Comment:
nit: you could have named it just "TestCase" (if django's TestCase was imported differently)